includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -26,6 +26,15 @@ public function __construct( $product, $productInstanceId, $lang = self::DEFAULT
 
 	public function getData() {
 		$data = [
+			'header' => [
+				'type' => 'link-image',
+				'image' => 'wds-company-logo-fandom-powered-by-wikia-two-lines',
+				'href' => 'http://fandom.wikia.com',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-footer-fandom-header'
+				]
+			],
 			'company_overview' => [
 				'header' => [
 					'type' => 'line-text',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "line-text" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)